cvat/apps/documentation/user_guide.md

```diff
@@ -362,7 +389,41 @@ Use shortcuts for undo/redo actions ``Ctrl+Z`` __/__  ``Ctrl+Shift+Z``/``Ctrl+Y`
 
 ![](static/documentation/images/image061.jpg)
 
+---
+__Fill Opacity slider__
+
+Changes opacity of every Bounding Box in the Annotation.
+
+![](static/documentation/images/image086.jpg)
+
+Opacity slides right to left, starting from ``white borders`` and ``100-80-60-40-20% white body`` to ``colored borders`` and ``0-20-40-60-80-100% colored body``.
+If any white option is chosen, Color By won't work.
+
+__Selected Fill Opacity slider__
+
+Changes opacity of Bounding Box under mouse pointer.
+
+![](static/documentation/images/image087.jpg)
+
+Opacity slides from 0% to 100% colored body
 
+__Black Stroke checkbox__
+
+Changes Bounding Box border from white/colored to black.
+
+![](static/documentation/images/image088.jpg)
+
+__Color By options__
+
+Changes color scheme of Annotation:
+- ``Instance`` — every Bounding Box have random color
+- ![](static/documentation/images/image095.jpg)
+- ``Group`` — every group of Boxes have its own random color, ungrouped Boxes are white
```

Comment:
Please don't use upper case in the middle of a sentence. What is a reason for that?